lib/generators/spotlight/install_generator.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable ClassLength
```

Comment:
e.g.

```
Metrics/ClassLength:
  Exclude:
    - 'lib/generators/spotlight/*'
```

Yeah, I don't know what the right separation is. Maybe revisit it later?